IssueComment {issueCommentUpdatedAt = GithubDate {fromGithubDate = 2013-05-01 20:02:33 UTC}, issueCommentUser = GithubUser {githubOwnerAvatarUrl = "https://secure.gravatar.com/avatar/272540e56a9b1b5c01fa5ef3c7a91edd?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png", githubOwnerLogin = "jseabold", githubOwnerUrl = "https://api.github.com/users/jseabold", githubOwnerId = 296164, githubOwnerGravatarId = "272540e56a9b1b5c01fa5ef3c7a91edd"}, issueCommentUrl = "https://api.github.com/repos/pydata/patsy/issues/comments/17303024", issueCommentCreatedAt = GithubDate {fromGithubDate = 2013-05-01 20:02:33 UTC}, issueCommentBody = "Just came to file this same request, so +1", issueCommentId = 17303024}